src/services/address/address.ts

```diff
@@ -20,4 +22,12 @@ const isHumanizerKnownTokenOrSmartContract = (
   )
 }
 
-export { isValidAddress, isHumanizerKnownTokenOrSmartContract }
+const formatNativeTokenAddressIfNeeded = (address: string) => {
+  if (getAddress(address) === '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE') {
+    return '0x0000000000000000000000000000000000000000'
+  }
+
+  return address
+}
```

Comment:
TODO (for me): this is probably no longer needed after normalizing the incoming tokens. Double-check before mergeing!

Reply:
Yep, removed in https://github.com/AmbireTech/ambire-common/pull/1005/commits/ac5f55c63b299f75da65108f3cee7a47e78c3bdd